test/dynamo/test_dynamic_shapes.py

```diff
@@ -26,7 +27,11 @@
 
 def make_dynamic_cls(cls):
     return make_test_cls_with_patches(
-        cls, "DynamicShapes", "_dynamic_shapes", ("dynamic_shapes", True)
+        cls=cls,
+        make_new_cls=True,
```

Comment:
when would `make_new_cls` ever be false 🤔

Reply:
It was done for the other commit in the stack. But now that you want me to generate all tests, let's just do that. Then we won't need to touch this either, so no review overhead.